docs/installation.rst

```diff
@@ -2,16 +2,10 @@
 Installation
 ============
 
-Prerequisites
--------------
-* python V3.6 or later.
-* for interactive use, an ansi (vt100-compatible) terminal application.  On Windows 10, the 
-  separately-installable `Windows Terminal app <https://github.com/microsoft/terminal/releases>`_ 
-  is recommended.
-  
 Installation
 ------------
-Xonsh can be installed via package manager, via appimage or just run from a pre-built container:
+
+Xonsh can be installed via different package managers from python specific to operation system package managers:
```

Comment:
probably, "Xonsh can be installed & run with various package managers, by using appimage, or from a docker container:"

Reply:
Changed. Thanks!